File: Java/Activity53.java
import java.util.Scanner;
import java.util.Arrays;

public class Activity53 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        double[] scores = new double[10];

        System.out.println("Please enter 10 scores:");

        for (int i = 0; i < 10; i++) {
            scores[i] = in.nextDouble();
        }

        Arrays.sort(scores);

        double sum = 0;
        for (int i = 1; i < 9; i++) {
            sum += scores[i];
        }

        double finalScore = sum / 8;

        System.out.printf("The final score is: %.2f\n", finalScore);
    }
}

File: C/Activity53.c
#include <stdio.h>

int main() {
    float scores[10];
    float sum = 0.0, final_score;
    float max = 0.0, min = 10.0;

    printf("Please enter 10 scores:\n");

    // Input scores
    for (int i = 0; i < 10; i++) {
        scanf("%f", &scores[i]);

        // Track max and min
        if (scores[i] > max) max = scores[i];
        if (scores[i] < min) min = scores[i];

        sum += scores[i];
    }

    // Remove highest and lowest score
    sum = sum - max - min;

    // Compute average of remaining 8 scores
    final_score = sum / 8.0;

    printf("The final score is: %.2f\n", final_score);

    return 0;
}
